src/mscorlib/src/System/String.cs

```diff
@@ -174,18 +174,28 @@ unsafe public char[] ToCharArray(int startIndex, int length)
             return Array.Empty<char>();
         }
 
-        public static bool IsNullOrEmpty(String value)
+        public static bool IsNullOrEmpty(string value)
         {
-            return (value == null || value.Length == 0);
+            return (value is null || value.Length == 0);
```

Comment:
Why are you changing such == to is? I personally dislike the style, as it leads to inconsistencies with other comparisons. Is there a functional difference here?

Reply:
`==` suggests it calls `operator ==`; whereas `is null` doesn't. I know string is special cased but it happens for other things.

Can switch back?